docs/content/latest/explore/ysql-language-features/gin.md

```diff
@@ -347,11 +345,6 @@ SubDocKey(DocKey([], ["\x05one", EncodedSubDocKey(DocKey(0xfca0, [3], []), [])])
 SubDocKey(DocKey([], ["\x05thing", EncodedSubDocKey(DocKey(0x9eaf, [4], []), [])]), [SystemColumnId(0); HT{ physical: 1636678107997627 w: 1 }]) -> null; intent doc ht: HT{ physical: 1636678107961628 }
 ```
 
-## Roadmap
-
-GIN indexes are still in progress: see the GIN roadmap tracking [GitHub issue][github-roadmap] for more details.
-
-[github-roadmap]: https://github.com/yugabyte/yugabyte-db/issues/7850
```

Comment:
Why is this deleted?  Even if you have a legitimate reason, it's out of character with the rest of the changes that it should either be done in a separate PR or called out in the commit message (along with the ybgin to gin update).

Reply:
This is just Explore section. Also all this should go in Github Master ticket.